keyboards/scarletbandana/config.h

```diff
@@ -54,10 +53,3 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #endif
 
```

Comment:
Should probably add the following, since LTO is enabled in the rules:
```suggestion

#define NO_ACTION_MACRO
#define NO_ACTION_FUNCTION

```